crates/re_types/definitions/rerun/blueprint/archetypes/visual_bounds.fbs

```diff
@@ -0,0 +1,23 @@
+include "arrow/attributes.fbs";
+include "python/attributes.fbs";
+include "rust/attributes.fbs";
+
+include "rerun/datatypes.fbs";
+include "rerun/attributes.fbs";
+
+namespace rerun.blueprint.archetypes;
+
+
+/// Controls the visual bounds of a 2D space view.
+table VisualBounds (
+    "attr.rerun.scope": "blueprint",
+    "attr.rust.derive": "Copy"
+) {
+    /// The visible parts of a 2D space view, in the coordinate space of the scene.
+    ///
+    /// Everything within these bounds are guaranteed to be visible.
+    /// Somethings outside of these bounds may also be visible due to letterboxing.
+    visual_bounds: rerun.components.AABB2D ("attr.rerun.component_optional", nullable, order: 1000);
+
+    // Here we'll add things like locking the visual bounds of one space view to another
```

Comment:
I doubt that we'll do this with special components - for this we'd rather establish links directly with more universal linking mechanisms.

What we do miss here is canvas cutout options: I'm arguing we regressed for the moment here because we no longer allow hiding things outside of your image